### PR TITLE
[AIP-132] List parent field should be required.

### PR DESCRIPTION
**Pull request by lukesneeringer**

### CODE REVIEW

File: docs/rules/0132/request-parent-behavior.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  summary: |
+    Get RPCs should annotate the `parent` field with `google.api.field_behavior`.
+permalink: /132/request-parent-behavior
+redirect_from:
+  - /0132/request-parent-behavior
+---
+
+# List methods: Field behavior
+
+This rule enforces that all `Get` standard methods have
```
Comment (mingzhi) [inline docs/rules/0132/request-parent-behavior.md]: Typo `Get`?

File: docs/rules/0132/request-parent-behavior.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+rule:
+  aip: 132
+  name: [core, '0132', request-parent-behavior]
+  summary: |
+    Get RPCs should annotate the `parent` field with `google.api.field_behavior`.
```
Comment (mingzhi) [inline docs/rules/0132/request-parent-behavior.md]: Typo `Get`?

File: docs/rules/0132/request-parent-behavior.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+# List methods: Field behavior
+
+This rule enforces that all `Get` standard methods have
+`google.api.field_behavior` set to `REQUIRED` on their `string parent` field,
+as mandated in [AIP-132][].
+
+## Details
+
+This rule looks at any message matching `List*Request` and complains if either
```
Comment (mingzhi) [inline docs/rules/0132/request-parent-behavior.md]: Remove "either"?

File: docs/rules/0132/request-parent-behavior.md (hunk 1 above)
Comment (lukesneeringer) [reply docs/rules/0132/request-parent-behavior.md]: Fixed, thanks!

File: docs/rules/0132/request-parent-behavior.md (hunk 2 above)
Comment (lukesneeringer) [reply docs/rules/0132/request-parent-behavior.md]: Fixed, thanks!

File: docs/rules/0132/request-parent-behavior.md (hunk 3 above)
Comment (lukesneeringer) [reply docs/rules/0132/request-parent-behavior.md]: Fixed, thanks!

**Pull request merged**